### PR TITLE
chore: refactor house module

### PR DESCRIPTION
**Pull request by 3eyed-raven**
## Description

---

## Type of change
Code Refactor

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code Refactor
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Added Features

- [x] NA

---

## Modified Features

- [x] House module

---

## Test Cases

- [ ] Test A
- [ ] Test B

---

## Change Log

---

## Documentation

---

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
---

### CODE REVIEW

File: proto/sge/house/query.proto [hunk 1]
```diff
@@ -19,19 +19,19 @@ service Query {
...
-    option (google.api.http).get = "/sge/deposits";
+    option (google.api.http).get = "/sge/house/deposits";
   }
 
-  // DepositorDeposits queries deposits info for given depositor.
-  rpc DepositorDeposits(QueryDepositorDepositsRequest)
-      returns (QueryDepositorDepositsResponse) {
-    option (google.api.http).get = "/sge/deposits/{depositor_address}";
+  // DepositsByUser queries deposits info for given user.
+  rpc DepositsByUser(QueryDepositsByUserRequest)
```
Comment (scorpioborn) [inline proto/sge/house/query.proto:26]: In Cosmos App, "user" is not a industry word, I think the `user` can be replaced with 'account' or `acc` or `depositor`

**Review by scorpioborn — CHANGES_REQUESTED** [review]

File: proto/sge/house/query.proto (hunk 1 above)
Comment (3eyed-raven) [reply proto/sge/house/query.proto]: Modified everywhere accordingly @scorpioborn

File: x/house/keeper/deposit.go [hunk 2]
```diff
@@ -31,9 +34,9 @@ func (k Keeper) GetDeposit(ctx sdk.Context, depositorAddress, marketUID string,
 	return val, true
 }
 
-// GetAllDeposits returns all deposits used during genesis dump.
-func (k Keeper) GetAllDeposits(ctx sdk.Context) (list []types.Deposit, err error) {
-	store := k.getDepositsStore(ctx)
+// GetDeposits returns returns list of deposits.
```
Comment (P-U-D-G-E) [inline x/house/keeper/deposit.go:37]: GetDeposits returns returns list of deposits. -> GetDeposits returns list of deposits.

**Review by P-U-D-G-E — CHANGES_REQUESTED** [review]

**Review by scorpioborn — DISMISSED** [review]

File: x/house/keeper/deposit.go (hunk 2 above)
Comment (3eyed-raven) [reply x/house/keeper/deposit.go:37]: Resolved

**Review by P-U-D-G-E — APPROVED** [review]

**Pull request merged**